src/App.js

```diff
@@ -492,88 +523,110 @@ class App extends Component {
     };
     return (
       !this.state.isAuthenticating && (
-        <Sentry.ErrorBoundary
-          // eslint-disable-next-line no-unused-vars
-          fallback={({ error, componentStack, resetError }) => (
-            <>
-              <div>
-                Dear user, you have (sadly) encountered an error. The error is
-                written out for you below, but it's probably useless to you. If
-                you are just interested in moving past this unfortunate
-                incident, click the button below to reload the page and start
-                fresh.
-              </div>
-              <div>{error.toString()}</div>
-              <div>{componentStack}</div>
-              <button onClick={() => window.location.replace("/")}>
-                Click here to reset!
-              </button>
-            </>
-          )}
-        >
-          <>
-            {this.state.isAuthenticated ? (
+        <ThemeProvider theme={theme}>
+          <Sentry.ErrorBoundary
+            // eslint-disable-next-line no-unused-vars
+            fallback={({ error, componentStack, resetError }) => (
               <>
-                <div className="sticky-logout" onClick={this.handleLogout}>
-                  <GrLogout />
-                </div>
-
-                <div className="root-padding">
-                  <LeftNav
-                    chosenLanguage={this.state.chosenLanguage}
-                    updateState={this.setState.bind(this)}
-                    user={this.state.user}
-                    athletes={this.state.athletes}
-                  />
-
-                  <Routes childProps={childProps} />
-                </div>
-                <div className="sticky-nav">
-                  <div className="sticky-chat">
-                    <Image
-                      src={question}
-                      onClick={(event) => {
-                        event.preventDefault();
-                        this.setState({ isTourOpen: true });
-                      }}
-                      height="65"
-                      width="65"
-                      circle
-                      className="sticky-btn"
-                      style={{ marginRight: 12, cursor: "pointer" }}
-                    />
-                  </div>
-                  <div id="myBottomNav" className="bottom-nav">
-                    <BottomNav user={this.state.user} />
-                  </div>
+                <div>
```

Comment:
Ticket Item: Update the default error catcher styling, add borders to the error message, center the text somewhat with margin all-around. Add Material Buttons for the reset buttons.

Reply:
👍 Added this to the queue.